src/main/java/uk/gov/hmcts/ccd/domain/model/draft/UpdateCaseDraftBuilder.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.ccd.domain.model.draft;
+
+public class UpdateCaseDraftBuilder {
+    private final UpdateCaseDraftRequest updateCaseDraftRequest;
```

Comment:
since we are writing many builders in this PR, perhaps is worth mentioning that the way we are implementing Builders here is not the recommended/standard way of doing it. 
The recommended way would be to have all the fields of the object we want to build as fields of the builder. And then to instantiate the object in the build method. This allows the creation of immutable objects.
Also, builders are usually worth the effort when the object to build is a complex object. Not sure i would provide a builder for a class with 3/4 fields only

Reply:
done